Control coverage/Feature/AppService.md

```diff
@@ -2,5 +2,8 @@
 
 | ControlId | Dependent Azure API(s) and Properties | Control spec-let |
 |-----------|-------------------------------------|------------------|
-| <b>ControlId:</b><br>Azure_AppService_Config_Disable_Remote_Debugging<br><b>DisplayName:</b><br>Remote debugging should be turned off for App Service.<br><b>Description: </b><br> Ensure that only the most secure and up to date version of TLS is enabled on the API gateway. |<b> ARM API to get configuration of an App Service: </b> <br> /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/config/web? <br> api-version=2018-11-01 <br><b>Properties:</b><br> properties/remoteDebuggingEnabled| <b>Passed: </b><br>Remote debugging for resource is turned OFF.<br><b>Failed: </b><br>Remote debugging for resource is turned ON. |
-| <b>ControlId:</b><br>Azure_AppService_Config_Disable_Web_Sockets<br><b>DisplayName:</b><br>Web Sockets should be disabled for App Service. <br><b>Description: </b><br> Web Sockets should be disabled for App Service. | <b> ARM API to get configuration of an App Service: </b> <br> /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/config/web?<br> api-version=2018-11-01 <br><b>Properties:</b><br> properties/webSocketsEnabled | <b>Passed: </b><br>Web sockets for resource  is disabled.<br><b>Failed: </b><br>Web sockets for resource  is enabled. |
\ No newline at end of file
+| <b>ControlId:</b><br>Azure_AppService_DP_Dont_Allow_HTTP_Access<br><b>DisplayName:</b><br>Use HTTPS for app services.<br><b>Description: </b><br> App Service must only be accessible over HTTPS. |<b> ARM API to list all the App Services in a subscription: </b> <br> /subscriptions/{subscriptionId}/providers/Microsoft.Web/sites? <br> 2018-11-01 <br><b>Properties:</b><br> properties/httpsOnly <br><br> <b> ARM API to list all deployment slots of an App Service: </b> <br> /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/slots? <br> api-version=2019-08-01 <br><b>Properties:</b><br> properties/httpsOnly| <b>Passed: </b><br>HTTPS Only setting is eanbled for all slots in App Service.<br><b>Failed: </b><br>HTTPS Only setting is disabled for any slot in App Service. |
```

Comment:
Typo in "Only setting is **eanbled** for all slots in App Service.". **eanbled** to be corrected to **enabled**.

Reply:
done, fixed the typo.